tests/scripts/mbedtls_test.py

```diff
@@ -163,6 +163,7 @@ def __init__(self):
         self.tests = []
         self.test_index = -1
         self.dep_index = 0
+        self.suite_result = True;
```

Comment:
If I understand correctly, the way this works is:

* `self.suite_result` contains the pass (True) or failed (False) status of the test suite
* After every test is completed, the function `on_result()` is called and it updates the `self.suite_result` to the correct status according to the result.

If this is the case, I suggest renaming it to make the member variable `self.suite_result` to something like `self.suite_passed` or something like that to make the boolean value self explanatory.

Reply:
>After every test is completed, the function on_result() is called and it updates the self.suite_result to the correct status according to the result.

Not exactly. If one of the tests in the suite failed, we set the value to `FALSE`. we never set it to `TRUE` if a test passed. When the test suite finished, we notify greentea client that the suite is finished, with the result status, whether it failed or passed.
I'll rename the variable.